server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const port = process.env.PUBLIC_PORT;
+
+// define the ping route
+app.get('/ping', (req,res)=>{
```

Comment:
Consider using a more descriptive endpoint name than '/ping'. It would be more meaningful if it reflects the functionality it provides. [medium]